app/views/help_page/show.html.erb

```diff
@@ -0,0 +1,9 @@
+<% content_for :extra_head_content do %>
```

Comment:
This should be `extra_headers` not `extra_head_content`. It's something that tripped up both myself and Keith.

You can check if the metadata is being displayed by viewing the page source. If the metaproperty appears it's all ok:

```
<meta property="og:description" content="How to upgrade your web browser - improve safety, speed and features">
<meta property="og:title" content="Browsers">
<meta property="og:url" content="https://www.gov.uk/help/browsers">
<meta property="og:type" content="article">
<meta property="og:site_name" content="GOV.UK">
<meta name="twitter:card" content="summary">
```
To combat this Keith started porting over the [meta tags tests](https://github.com/alphagov/frontend/pull/4218/commits/975dca796779ef0a3ec0b5c3d836488164371e3e) as shared examples that could be added to all of the system tests. Perhaps that could be done in this PR instead?